### ISSUE TITLE
Python `setup.py` installation and GitHub releases

### ISSUE THREAD

**Issue by guoyunhe** [daehruoydeef/Yin-Yang#3]
Hi, I see this is a really promising project. I wonder if I can package it for openSUSE/SLE. Usually Python applications have `setup.py` as installation script. It makes packaging easy. Also GitHub releases is also useful for packaging. Package maintainers can watch releases to update their packages.

Thanks for the great work!

**Comment by neverix** [comment]
It is!
AUR needs this. I'll try to package it in the meantime.

**Comment by daehruoydeef** [comment]
Hello thanks for the effort. The setup.py is misleading and will be renamed to "setupUi.py" since it converts the QT5 ui form to pyqt bindings. It is not a setup.py in a pythonic way someone expect. I tried packaging this as a pip package with setup.py but failed miserable, any help would be much appreciated. I will try to create a setup.py again and see if it works.

**Comment by neverix** [comment]
Done! There's now an AUR package.
Waiting for setup.py now (this will ease the installation).

**Comment by neverix** [comment]
Any updates on this?

**Comment by daehruoydeef** [comment]
I tried to package this with pyinstaller but ran into Problems with PyQt, I am still trying to package this as a flatpak or to begin with in any one file executable. Any community help is much appreciated in order to get a proper format out with Yin-Yang 3.0 which should be released on 10.4

**Comment by daehruoydeef** [comment]
@guoyunhe @neverix 
good news on this, I managed to package it as a one-file-executable which basically is the same as a appimage however it does not feature to create a .desktop file neither a correct Desktop entry. I created this using PyInstaller. In order for correct installation I still would like to go the package manager route and package this up for each distribution. Right now I can only offer the Yin-Yang singlefile executable without any installer.
